nixos/modules/misc/mandoc.nix

```
@@ -41,6 +44,25 @@ in
         '';
       };
 
+      cachePath = lib.mkOption {
+        type = with lib.types; listOf str;
+        default = cfg.manPath;
+        defaultText = lib.literalExpression "config.documentation.man.mandoc.manPath";
```

Don't we want the following?

```suggestion
        default = cfg.manPath ++ cfg.settings.manpath;
        defaultText = lib.literalExpression "config.documentation.man.mandoc.manPath ++ config.documentation.man.mandoc.settings.manpath";
```

---

We can't. We are generating mandoc.db when building /run/current-system. This is required because mandoc will only search for it in the same path of manpages.

---

Right.